library/src/scala/reflect/Selectable.scala

```diff
@@ -0,0 +1,73 @@
+package scala.reflect
+
+class Selectable(val receiver: Any) extends AnyVal with scala.Selectable {
```

Comment:
I would call this `ReflectiveSelectable` or something like that, it's easy to miss the prefix.